addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -130,76 +138,69 @@ function TOOL:GetNormalSpawn(stTr, eEnt)
   return vNorm, aNorm
 end
 
+function TOOL:CheckOwnUID(sUID, uNtf, bJM)
```

Comment:
This is here to check for ownership instead calling the loop two times in a row on exact same UID